### ISSUE TITLE
npm publish fails

### ISSUE THREAD

**Issue by zeke** [jdxcode/npm-register#31]
It would be nice if this registry could forward/proxy publish commands to the main registry. 

Otherwise it looks like I'll have to change my npm config every time I want to publish something.

```
Command failed: npm publish
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! code ENEEDAUTH

npm ERR! need auth auth required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/zeke/zeke/electron-docs-linter/npm-debug.log
```

**Comment by okeydoke** [comment]
Adding an `.npmrc` file to the root of your project with something like the following might help
```
registry=https://my-thing.herokuapp.com
```
for scoped packages you can also add
```
@scope:registry=https://my-thing.herokuapp.com
```

**Comment by dickeyxxx** [comment]
This is a slightly different use-case than what I designed npm-register to be—to have private packages as well as public ones.

I'd be willing to make this configurable somehow. Maybe also with the configuration to solve #1. I'm not sure what that would look like right now though.

**Comment by dgautsch** [comment]
Seems this was addressed in #62 . Closing out.
